backend/src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:${CLEARDB_DATABASE_URL}
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.security.oauth2.resourceserver.jwt.issuer-uri = https://rota-flex-101.eu.auth0.com/
+auth0.audience = rota-flex-api
```

Comment:
These are repeated in application.properties. Is this intentional?

Reply:
Not intentional!